test/plugins/sentry.test.js

```diff
@@ -1,10 +1,8 @@
 const Raven = require('raven')
-
-const plugin = require('../../src/plugins/sentry')
-
 const helper = require('./helper')
+const appFn = require('../../src/plugins/sentry')
 
-describe('sentry', () => {
+describe('sentry app', () => {
```

Comment:
I'm also conflicted about our use of plugin/app terminology around sentry. Sentry feels more like an add-on service for an app, which makes me think maybe the plugin terminology makes sense here.